crates/turborepo-scm/src/git.rs

```diff
@@ -102,9 +97,9 @@ fn add_files_from_stdout(
 ) {
     let stdout = String::from_utf8(stdout).unwrap();
     for line in stdout.lines() {
-        let path = Path::new(line);
+        let path: RelativeUnixPath = line.into();
```

Comment:
Do we know that git2 always outputs unix paths? Even so, this should probably be explicit as an unsafe conversion.

Reply:
I don't know about `git2`. The `git` cli always uses unix paths.